README.md

```diff
@@ -1 +1,15 @@
-# tk-2022
\ No newline at end of file
+# Component based local filesystem image finder
+The puprose of this app is to integrate many open source components into one REST based application that allows to browse local directories for images that fullfil certain criteria. The app will be distributed along a few instances that share the load of image processing and communicate via REST (e.g. split subdirectories between the running instances). Besides the backend search architecture app also features a web based interface for defining the search criteria and displaying the results. The project is developed for the "Techniki Komponentowe" class at AGH Uuniversity of Science and Technology.
```

Comment:
"Component technologies"

Reply:
Won't fix

Reply:
This borders the territory of localization therefore it's fine to be left in the original form. (Since there is no official localization that I know of I would rather leave it untranslated.)